### PR TITLE
Feature: Stats page

### PR DESCRIPTION
**Pull request by VitroidFPV**
image
image

### CODE REVIEW

**Comment by haslinghuis** [comment]
image

**Comment by McGiverGim** [comment]
Why only the top 3 targets? I think a bigger list will be interesting. Three are too few.

**Comment by VitroidFPV** [comment]
More than 3 looked busy, but I can try putting it higher and posting the results to compare

**Comment by KarateBrot** [comment]
Can you encapsulate the plotting function so that we can use plots for any data? If that's too much, I can have a go at it in a later PR.

**Comment by VitroidFPV** [comment]
@KarateBrot the plots are just components from the Nivo library. You can use it anywhere else, you only need to feed it x and y labels

**Comment by VitroidFPV** [comment]
@McGiverGim here's a test with 5, I tried to compact the line down a bit to make it more readable
image

Any more and the targets don't really fit into the legend

**Comment by McGiverGim** [comment]
Ok better, thanks. Maybe a table or a sectorial graph for a future PR will admit more targets, with the aggregate data of last week for example.

**Comment by KarateBrot** [comment]
> @VitroidFPV the plots are just components from the Nivo library. You can use it anywhere else, you only need to feed it x and y labels

So I can use these plots in `.mdx` files?

Small design thingy: Can you add
- vertical lines to the left and right side of the plotting area
- ticks for the axes

File: src/pages/stats.tsx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  volume: number
+}
+
+async function getStats() {
+  const stats = await fetch(`https://build.betaflight.com/api/stats`).then((res) => res.json());
+  const data = stats.volumes.map((volume: Volume) => ({
+    x: new Date(volume.date).toLocaleDateString('en-GB', { day: 'numeric', month: 'short' }),
+    y: volume.cached,
+  }));
+  data.reverse();
```
Comment (freasy) [inline src/pages/stats.tsx:36]:
```suggestion
    x: new Date(volume.date).toLocaleDateString('en-GB', { day: 'numeric', month: 'short' }),
    y: volume.cached,
  })).reverse();
```

**Review by freasy — APPROVED** [review]
Code looks good to me! :)

**Review by sugaarK — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
@VitroidFPV there seems to be some issues with the PR:

https://github.com/betaflight/betaflight.com/actions/runs/5410123424

**Pull request merged**